### PR TITLE
Add new model for grouping emails

### PR DESCRIPTION
**Pull request by feeds**
This leaves the guid field as described in #350 on <null>

Migration by hand would be possible ;)

### CODE REVIEW

**Comment by feeds** [comment]
Closes #350

File: backend/apps/iamstudent/views.py [hunk 1]
```diff
@@ -77,9 +77,12 @@ def send_mail_student_id_list(request, id_list):
 
             hospital_message = form.cleaned_data['message']
 
+            subject = form.cleaned_data['subject']
 
+            email_group = EmailGroup.objects.create(subject=subject,
+                                                    message=hospital_message,
+                                                    sender=request.user.hospital)
```
Comment (maltezacharias) [inline backend/apps/iamstudent/views.py]: kwarg sender müsste hospital sein

**Review by maltezacharias — DISMISSED** [review]
Mismatch sender bzw. hospital als Argument für eMail Group crasht

File: backend/apps/iamstudent/views.py (hunk 1 above)
Comment (feeds) [reply backend/apps/iamstudent/views.py]: solved this in new commit

**Review by josauder — APPROVED** [review]

**Pull request merged**